.gitignore

```diff
@@ -164,3 +164,4 @@ backend/src/**/*.js
 backend/scripts/**/*.js
 backend/test/**/*.js
 backend/config.json
+claymore-frontend/package-lock.json
```

Comment:
Shouldn't we be committing package lock so we can reproduce environments between each of us (and also on the server)?

Reply:
added package-lock.json